__tests__/__util__/helpers/parsers.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const path = require('path');
+const semver = require('semver');
+const entries = require('object.entries');
+const version = require('eslint/package.json').version;
+const flatMap = require('array.prototype.flatmap');
+const tsParserVersion = require('@typescript-eslint/parser/package.json').version;
+
+const disableNewTS = semver.satisfies(tsParserVersion, '>= 4.1') // this rule is not useful on v4.1+ of the TS parser
+  ? (x) => Object.assign({}, x, { features: [].concat(x.features, 'no-ts-new') })
```

Comment:
```suggestion
  ? (x) => { ...x, features: [].concat(x.features, 'no-ts-new') }
```
since in this repo we have a build process :-)